packages/api/src/StyleOptions.ts

```diff
@@ -95,6 +95,11 @@ type StyleOptions = {
    */
   botAvatarInitials?: string;
 
+  /**
+   * Bot title
+   */
+  botTitle?: string;
```

Comment:
I think we should grab it from `activity.from.name`.

Reply:
Done, this is a bit strange as the rest (like bot image, backdrop color) comes from `styleOptions`. I hope there won't be a case when it is missing from the activity.